src/ado/reproot_setup.ado

```diff
@@ -51,23 +51,29 @@ qui {
     syntax, env_file(string) home_fld(string) [envpaths(string)]
 
     * Ask for confirmation
-    noi di as result _n `"{pstd}No environment file was found in the home folder [`home_fld']. This command can set up a template. In the workflow of {inp:reproot}, this file must be located in the home folder.{p_end}"' _n(2) `"{pstd}Do you want to set up a template in the home folder?{p_end}"'
+    noi di as result _n `"{pstd}No environment file was found in {browse `home_fld':your home folder}.{p_end}"' _n `"{pstd}Do you want to set up a template?{p_end}"'
     global setup_confirmation ""
     while (!inlist(upper("${setup_confirmation}"),"Y", "N")) {
-      noi di as txt `"{pstd}Enter "Y" to set up an environment template file or enter "N" to abort."', _request(setup_confirmation)
+      noi di as txt `"{pstd}Enter "Y" to continue or "N" to exit to Stata."', _request(setup_confirmation)
     }
     if upper("${setup_confirmation}") == "N" {
       noi di as txt "{pstd}Environment file creation aborted - nothing was created.{p_end}"
       error 1
       exit
     }
 
-    noi di as result _n `"{pstd}The environment file will only work if it has at least one environment search paths. Read more about the environment file and its settings in the {browse "https://worldbank.github.io/repkit/articles/reproot-files.html":this guide}.{p_end}"' _n(2) `"{pstd}Do you want to add search folders? It is possible to do this later see the guide (also linked to in the help file for this command) for how to modify this file later.{p_end}"'
+    noi di as result _n `"{pstd}Reproot requries at least one search path. Read more {browse "https://worldbank.github.io/repkit/articles/reproot-files.html":here}.{p_end}"' ///
+      _n `"{pstd}Do you want to add one now?{p_end}"' _n `"{pstd}See {help reproot} or {browse "https://worldbank.github.io/repkit/articles/reproot-files.html":this guide} for more information.{p_end}"' ///
+      _n `"{pstd}Some top-level options are:{p_end}"' _n `"{pstd}`home_fld'/...{p_end}"'
```

Comment:
```
_n `"{pstd}Some top-level options are:{p_end}"' _n `"{pstd}`home_fld'/...{p_end}"'
```
@bbdaniels , did you intend for examples to be listed here? My fear is that subset of users that needs this hint has a lot of overlap with the subset of users that will be confused by a folder path `/...` and will try to use exactly that.

Reply:
Suggesting removing all `home_fld/...` in the rest of this commit

Reply:
Yes, see above

Reply:
removed in e273bfbe1af98427c34cb464cac4b056a4ca26ac